qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy

```diff
@@ -135,6 +135,7 @@ class ProcessBaselinesTest extends BaseSpecification {
 
     def cleanupSpec() {
         PolicyService.deletePolicy(unauthorizedProcessExecution.getId())
+        orchestrator.deleteNamespace(TEST_NAMESPACE)
```

Comment:
This missing NS delete showed up as a backtrace while debugging this PR. Strictly speaking it should have been added when `ProcessBaselinesTest` was parallelized (#7474) and surfaces an issue that `compareResourcesAtRunEnd()` does not fail the test now that it is executed in the shutdown handler.